app/templates/base.html

```diff
@@ -19,7 +19,7 @@
       gtag('config', 'YOUR_TRACKING_ID');
   </script>
     <meta name="description" content="{{config['SERVICE_NAME']}}">
-    <meta name="keywords" content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, jinja, python, flask, port, template, templating, macro, component, design system, html, forms, wtf, wtforms, widget, widgets, demo, example">
+    <meta name="keywords" content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, jinja, python, flask, port, template, templating, macro, component, design system, html, forms, wtf, wtforms, widget, widgets, example">
```

Comment:
Can we remove all the keywords that don't relate to the service i.e. frontend onwards - unless there's a reason to keep them?

Reply:
Hiya Jav, this is just the base template, once we start making the service this will come out anyway